### PR TITLE
Handle Telemetry Empty results

### LINKED ISSUE #121: Telemetry query_result None should return incomplete and retry a bit after.

### PR DESCRIPTION
**Pull request by Natim**
Fixes #121

### CODE REVIEW

File: pollbot/tasks/telemetry.py [hunk 1]
```diff
@@ -125,6 +125,11 @@ def get_telemetry_auth_header():
             # In that case the query already exists
             # Get the results
             latest_query_data_id = query_info["latest_query_data_id"]
+
+            if latest_query_data_id is None:
+                url = "{}/queries/{}".format(TELEMETRY_SERVER, query_info['id'])
+                return build_task_response(Status.INCOMPLETE, url, "Query still processing.")
```
Comment (leplatrem) [inline pollbot/tasks/telemetry.py]: Maybe a detailed comment would help here (the above comment seems outdated `«Get the results»`)

File: tests/test_tasks.py [hunk 2]
```diff
@@ -887,6 +887,69 @@ def mock_platforms(self, platforms, working_body):
             await telemetry.update_parquet_uptake('firefox', '57.0a1')
         assert str(excinfo.value) == 'Query 40223 unavailable (HTTP 502)'
 
+    async def test_telemetry_update_uptake_tasks_returns_error_for_no_result(self):
```
Comment (leplatrem) [inline tests/test_tasks.py]: returns "incomplete" ?

File: tests/test_tasks.py [hunk 3]
```diff
@@ -887,6 +887,69 @@ def mock_platforms(self, platforms, working_body):
...
+        self.mocked.get(url, status=200, body=json.dumps({
+            "query_result": {"data": {"rows": [
+                {"build_id": "20170920220431"},
+                {"build_id": "20170920111019"},
+                {"build_id": "20170920100426"},
+                {"build_id": "20170919220202"},
+                {"build_id": "20170919110626"}
+            ]}}
+        }))
+
```
Comment (leplatrem) [inline tests/test_tasks.py]: All the above seems to be identical with the previous one. Having a common `setUp()` would help (maybe)

Comment (Natim) [reply tests/test_tasks.py]: I opened https://github.com/mozilla/PollBot/issues/127 to discuss that.

**Pull request merged**